src/services/consent/utils/aggregations.js

```diff
@@ -367,17 +394,16 @@ const createMultiDocumentAggregation = ({
 		stageBaseFilter(aggregation, 'classes', classes);
 	}
 
-
 	if (sort) {
 		stageSort(aggregation, sort);
 	}
 
-	if (selectSortDiff.length !== 0) {
-		stageSimpleProject(aggregation, select);
-	}
+	// if (selectSortDiff.length !== 0) {
+	// TODO: think about doing it after limit and skip
+	stageSimpleProject(aggregation, select);
+	// }
```

Comment:
@CordlessWool can you explain whats going on here?

Reply:
could be renamed to simplifyProject? it removes all attributes that are not needed for the output like sorting attributes